code/campaignclassic/src/main/java/com/adobe/marketing/mobile/CarouselPushTemplate.java

```diff
@@ -14,11 +14,12 @@
 import com.adobe.marketing.mobile.util.DataReader;
 import com.adobe.marketing.mobile.util.DataReaderException;
 import com.adobe.marketing.mobile.util.StringUtils;
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
-class CarouselPushTemplate extends AEPPushTemplate {
+class CarouselPushTemplate extends AEPPushTemplate implements Serializable {
```

Comment:
Same question about the reason these need to be Serializable.

Reply:
reason is in https://github.com/adobe/aepsdk-campaignclassic-android/pull/30#discussion_r1425810043